src/main/java/org/ideoholic/curium/model/documents/action/DocumentActionAdapter.java

```diff
@@ -123,4 +125,74 @@ public boolean transferCertificate() {
+		transferCertificateDto.setPinFig(request.getParameter("pinfig"));
+		transferCertificateDto.setPinWord(request.getParameter("pinword"));
+		transferCertificateDto.setDues(request.getParameter("dues"));
+		transferCertificateDto.setConcession(request.getParameter("concession"));
+		transferCertificateDto.setWorkingDays(request.getParameter("workingdays"));
+		transferCertificateDto.setPresent(request.getParameter("present"));
+		transferCertificateDto.setNcc(request.getParameter("ncc"));
+		transferCertificateDto.setGame(request.getParameter("game"));
+		transferCertificateDto.setConduct(request.getParameter("conduct"));
+		transferCertificateDto.setDateCert(request.getParameter("datecert"));
+		transferCertificateDto.setRemarks(request.getParameter("Remarks"));
+		transferCertificateDto.setDateOfTc(request.getParameter("dateoftc"));
+		TransferCertificateResponseDto transferCertificateResponseDto = documentService.generateTransferCertificate(transferCertificateDto);
+		
+		request.setAttribute("leavingReason", transferCertificateResponseDto.getReason());
+		request.setAttribute("bookno", transferCertificateResponseDto.getBookNo());
+		 request.setAttribute("tcno", transferCertificateResponseDto.getTcNo());
+		request.setAttribute("caste", transferCertificateResponseDto.getCaste());
+		request.setAttribute("classinword", transferCertificateResponseDto.getClassInWord());
+		request.setAttribute("lastexam", transferCertificateResponseDto.getLastExam());
+		request.setAttribute("failpass", transferCertificateResponseDto.getFailPass());
+		request.setAttribute("firstsubject", transferCertificateResponseDto.getFirstSubject());
+		request.setAttribute("secondsubject", transferCertificateResponseDto.getSecondSubject());
+		request.setAttribute("thirdsubject", transferCertificateResponseDto.getThirdSubject());
+		request.setAttribute("Fourthsubject",transferCertificateResponseDto.getFourthSubject());
+		request.setAttribute("Fifthsubject", transferCertificateResponseDto.getFifthSubject());
+		request.setAttribute("sixthsubject", transferCertificateResponseDto.getSixthSubject());
+		request.setAttribute("pinfig", transferCertificateResponseDto.getPinFig());
+		request.setAttribute("pinword", transferCertificateResponseDto.getPinWord());
+		request.setAttribute("dues", transferCertificateResponseDto.getDues());
+		request.setAttribute("concession",transferCertificateResponseDto.getConcession());
+		request.setAttribute("workingdays",transferCertificateResponseDto.getWorkingDays());
+		request.setAttribute("present", transferCertificateResponseDto.getPresent());
+		request.setAttribute("ncc",transferCertificateResponseDto.getNcc());
+		request.setAttribute("game", transferCertificateResponseDto.getGame());
+		request.setAttribute("conduct", transferCertificateResponseDto.getConduct());
+		request.setAttribute("datecert", transferCertificateResponseDto.getDateCert());
+		request.setAttribute("Remarks", transferCertificateResponseDto.getRemarks());
```

Comment:
Also need to add:
`request.setAttribute("dateinword", transferCertificateResponseDto.getDateInWord());`